client/src/components/ForgotPassword/CheckEmail/CheckEmailContainer.tsx

```diff
@@ -1,49 +1,42 @@
 import React from "react";
+import CheckEmail from "./CheckEmail";
+import { reduxForm } from "redux-form";
 import { connect } from "react-redux";
-import CheckEmail from './CheckEmail';
-import {
-    setEmail,
-    checkEmail
-} from "../../../store/checkEmail/action";
+import { checkEmail } from "../../../store/checkEmail/actions";
 
-interface FormProps {
-    email: string;
-    setEmail: Function;
-    checkEmail: Function;
-}
-
-interface CheckEmailState {
-    email: string;
-}
+const validate = (user: any) => {
+  const errors: any = {};
+  if (!user.email) {
+    errors.email = "Please, enter your email!";
+  } else if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(user.email)) {
+    errors.email = "Invalid email address";
+  }
+  return errors;
+};
 
-interface FormState {
-    checkEmail: CheckEmailState;
-}
+class CheckEmailContainer extends React.Component<any> {
```

Comment:
maybe 
```
export interface IEmailObj {
   email: string;
}

export interface IProps {
   handleSubmit: (param: IEmailObj) => void;
   submitting: boolean;
```
something like this

Reply:
Ostap will take care of types in our front-end.